README.md

```diff
@@ -1,34 +1,85 @@
+- [จุดเริ่มต้นของ CU Get Reg](#beginning-of-cu-get-reg)
+- [โครงการที่เกี่ยวข้อง](#related-repositories)
+- [มาเป็นส่วนหนึ่งกับเรา](#contribution)
+- [Roadmap](#roadmap)
+- [ทีมผู้พัฒนา](#developers)
+- [ติดต่อเรา](#contacts)
+- [ผู้สนับสนุน](#sponsers)
 
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+<div id="beginning-of-cu-get-reg"></div>
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+## จุดเริ่มต้นของ CU Get Reg
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+    "GenEd เทอมนี้เราจะลงวิชาอะไรดีนะ?"
 
-## Learn More
+    "GenEd ที่เราเลือกลงไปเวลาจะไปทับกับวิชาบังคับหรือเปล่านะ?"
 
-To learn more about Next.js, take a look at the following resources:
+    "จัดตารางเรียนยังไรดีนะ? วันเรียนในแต่ละ Section ก็ไม่เหมือนกันอีก สับสนมาก ๆ"
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+และอีกสารพัดคำถามมากมายท่ามกลางความสับสนว่าจะวางแผนการเรียนอย่างไรดีในภาคการศึกษานี้ แน่นอนว่า [ทีมผู้พัฒนา](#developers) ก็เป็นหนึ่งในนั้น
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+การวางแผนลงทะเบียนในแต่ละเทอมเมื่อก่อนยุ่งยากและเสียเวลามาก ๆ ข้อมูลวิชาเรียนกระจัดกระจาย โดยเฉพาะอย่างยิ่งวิชา GenEd การจะหาวิชา GenEd ที่ใช่ เวลาเรียนที่ลงตัว ต้องหาข้อมูลจากหลากหลายแหล่งเพื่อประกอบการตัดสินใจ อีกทั้งยังมีเรื่องที่นั่งและความนิยมของวิชา GenEd เป็นปัจจัยสำคัญต่อการตัดสินใจ แต่ข้อมูลเหล่านั้นกลับเข้าถึงได้ยาก ทำให้วางแผนได้ยากลำบาก
 
-## Deploy on Vercel
+[ทีมผู้พัฒนา](#developers) เห็นถึงปัญหานี้ จึงได้วางแผนจัดทำเว็บไซต์เพื่ออำนวยความสะดวกในการวางแผนลงทะเบียนขึ้นมา
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/import?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
+<div id="related-repositories"></div>
 
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+## โปรเจคที่เกี่ยวข้อง
+
+- [cugetreg-frontend](https://github.com/thinc-org/cugetreg-frontend) เว็บไซต์ CU Get Reg
+- [cugetreg-backend](https://github.com/thinc-org/cugetreg-backend) ระบบหลังบ้านของเว็บไซต์ CU Get Reg
+- [cugetreg-scraper](https://github.com/thinc-org/cugetreg-scraper) ระบบคัดลอกข้อมูลวิชาเรียนแบบสาธารณะจาก [Reg Chula](https://cas.reg.chula.ac.th/cu/cs/QueryCourseScheduleNew/index.html)
+- [chula-courses](https://github.com/thinc-org/chula-courses) Schema ส่วนกลางของข้อมูลรายวิชา
```

Comment:
both of these repos are private

Reply:
Will we publicize the repos?

Reply:
แอบรู้สึกว่า `chula-courses` คนจะงงไหมนะว่าเราเก็บแค่ type ไม่มี function ไว้เรียก data แต่อย่างใด

`cugetreg-scraper` is probably ok to go public